data/json/npcs/factions.json

```diff
@@ -579,5 +579,84 @@
       "marloss": { "kill on sight": true }
     },
     "description": "A small group of churchgoers that formed a community in the woods.  They welcome anyone in their faction, but hate the unnatural."
+  },
+  {
+    "type": "faction",
+    "id": "lapin",
+    "name": "Rabbit Enthusiasts",
+    "likes_u": 30,
+    "respects_u": 20,
+    "known_by_u": false,
+    "size": 1,
+    "power": 1,
+    "currency": "fur",
+    "food_supply": 87500,
+    "wealth": 82500,
+    "relations": {
+      "free_merchants": {
+        "kill on sight": false,
+        "watch your back": false,
+        "share my stuff": false,
+        "guard your stuff": false,
+        "lets you in": true,
+        "defends your space": true,
+        "knows your voice": true
+      },
+      "old_guard": {
+        "kill on sight": false,
+        "watch your back": false,
+        "share my stuff": false,
+        "guard your stuff": false,
+        "lets you in": false,
+        "defends your space": false,
+        "knows your voice": false
+      },
+      "tacoma_commune": {
+        "kill on sight": false,
+        "watch your back": true,
+        "share my stuff": true,
```

Comment:
He's that much of an ally with the Tacoma Commune that they'll let him use their stuff without complaint?  Why?